css/webstyle.css

```diff
@@ -0,0 +1,330 @@
+body {
+  background: #e5e5e5;
+  font-family: "Poppins", sans-serif;
+  display: flex;
+  flex-direction: column;
+  margin: auto;
+}
+
+header {
+  display: flex;
+  position: sticky;
+  top: 0;
+  height: 7.86vh;
+  background-color: #fff;
+  align-items: baseline;
+}
+
+h3 {
+  flex: 1;
+  margin-left: 10%;
+  color: #6070ff;
+}
+
```

Comment:
- [ ] Please use media queries for the Desktop breakpoint in the CSS file. Since your styles for the Desktop screen-size are in `css/webstyle.css` file. I suggest you put all these styles within a media query. Remember to have the value for min-width to be in the range of 992px - 1100px).

Reply:
Where you get requirements [ min-width to be in the range of 992px - 1100px] because in our task was no word about some numbers. But for you :) I included media queries with min and also max values inside my `css/webstyle.css` applied to the main section's background.  
But anyway thanks for your review, it just improved my knowledge and got even more practice 👍